### DISCUSSION TITLE
More comprehensive Latex in MarkDown for Physics and Quantum Computing/ KaTex replacement or enhancement

### DISCUSSION THREAD

**Discussion by osbama** [Zettlr/Zettlr#3529] [Ideas]
Unfortunately, the latex implementation used in Zettlr (KaTex) is pretty limited for use in the Physics, Chemistry, and mathematics of Quantum computing. I think this is a crippling factor for those of us that want to use it for basic science note taking. It would be great if there was a choice for choosing the Latex rendering backend or enhance KaTex in a way that allows

- CTAN Package support (such as "physics" and "Quantikz") 
- Global/Local latex macro support.

**Comment by GeoffClements** [comment]
Indeed. I have just investigated using Zettlr as an alternative to Obsidian (some three years on after the OP's comment), and I absolutely must have `physics` and `siunitx` at a minimum. This one issue is preventing me from moving.
